From eab99c1330f572a25f6f6b271d412a8140ea0e8f Mon Sep 17 00:00:00 2001
From: "Wei-Cheng Pan (潘韋成)" <legnaleurc@gmail.com>
Date: Tue, 11 Aug 2015 12:37:56 +0000
Subject: [PATCH] Fix hard-coded URL handlers.

---
 data/lilyterm.conf | 8 ++++----
 src/profile.c      | 8 ++++----
 2 files changed, 8 insertions(+), 8 deletions(-)

diff --git a/data/lilyterm.conf b/data/lilyterm.conf
index c93bb16..30de889 100644
--- a/data/lilyterm.conf
+++ b/data/lilyterm.conf
@@ -567,28 +567,28 @@ Color15 =
 # Left it blank to use the locale environs from current page.
 
 # The web browser using for http(s)://
-web_browser = firefox
+web_browser = xdg-open
 web_method = 1
 web_VTE_CJK_WIDTH = 0
 web_environ = 
 web_locale = 
 
 # The ftp client using for ftp(s)://
-ftp_client = firefox
+ftp_client = xdg-open
 ftp_method = 1
 ftp_VTE_CJK_WIDTH = 0
 ftp_environ = 
 ftp_locale = 
 
 # The file manager using for file:// and [Open current directory with file manager]
-file_manager = firefox
+file_manager = xdg-open
 file_method = 1
 file_VTE_CJK_WIDTH = 0
 file_environ = 
 file_locale = 
 
 # The email client using for user@host
-email_client = thunderbird
+email_client = xdg-open
 email_method = 1
 email_VTE_CJK_WIDTH = 0
 email_environ = 
diff --git a/src/profile.c b/src/profile.c
index aee465f..e79e9a2 100644
--- a/src/profile.c
+++ b/src/profile.c
@@ -255,10 +255,10 @@ void init_user_command(struct Window *win_data)
 		if (win_data==NULL) return;
 #endif
 
-	win_data->user_command[TAG_WWW].command = g_strdup("firefox");
-	win_data->user_command[TAG_FTP].command = g_strdup("firefox");
-	win_data->user_command[TAG_FILE].command = g_strdup("firefox");
-	win_data->user_command[TAG_MAIL].command = g_strdup("thunderbird");
+	win_data->user_command[TAG_WWW].command = g_strdup("xdg-open");
+	win_data->user_command[TAG_FTP].command = g_strdup("xdg-open");
+	win_data->user_command[TAG_FILE].command = g_strdup("xdg-open");
+	win_data->user_command[TAG_MAIL].command = g_strdup("xdg-open");
 
 	gint i;
 	for (i=0; i<COMMAND; i++)
-- 
2.1.4

