Description: Fix wrong locale file name
Bug-Debian: http://bugs.debian.org/757080
---
Bug-Debian: http://bugs.debian.org/757080
Forwarded: https://github.com/Tetralet/LilyTerm/issues/84
--- lilyterm-0.9.9.4.orig/configure.ac
+++ lilyterm-0.9.9.4/configure.ac
@@ -51,7 +51,7 @@ AC_DEFINE([BUGREPORT], ["$_BUGREPORT"],
 
 AC_PROG_INTLTOOL
 ALL_LINGUAS="$_LANG_LIST"
-GETTEXT_PACKAGE="$_PACKAGE"
+GETTEXT_PACKAGE="$_BINARY"
 AC_SUBST(GETTEXT_PACKAGE)
 AM_GLIB_GNU_GETTEXT
 AM_GLIB_DEFINE_LOCALEDIR(LOCALEDIR)
